packages/transmute-adapter-orbit-db/package.json

```diff
@@ -4,17 +4,20 @@
   "description": "Simple bytes32 compatible content addressed json object interface for orbit db.",
   "main": "./src/index.js",
   "scripts": {
-    "test": "jest ",
-    "coverage": "jest --coverage"
+    "test": "jest",
+    "coverage": "npm run test --coverage"
```

Comment:
This does not work, it will ignore the --coverage flag. Replace by jest --coverage